explore-analyze/ai-assistant/manage-access-to-ai-assistant.md

```diff
@@ -0,0 +1,73 @@
+---
+applies_to:
+  stack: ga 9.2
+  serverless: ga
+products:
+  - id: kibana
+  - id: observability
+  - id: security
+  - id: cloud-serverless
+---
+
+# Manage access to AI Assistant
+
+This page describes how to use the GenAI Settings page to control access to AI Assistant in your deployments in the following ways:
+
+::::{applies-switch}
+
+:::{applies-item} stack: ga 9.2
+
+- Specify that a specific LLM connector must be used for conversations with AI Assistant. 
+- {applies_to}`stack: ga 9.2` {applies_to}`serverless: unavailable` Specify in which Elastic solutions the `AI Assistant for Observability and Search` and the `AI Assistant for Security` appear.
+- Disable AI Assistant access completely.
+
+:::
+
+:::{applies-item} serverless:
+
+- Manage which AI connectors are available in your environment. 
+- Enable or disable AI Assistant and other AI-powered features in your environment.
+
+:::
+
+::::
```

Comment:
Ideally the intro would be more generic and not need tabs. It doesn't have to be so precise at this point. 

- Turn Elastic AI Assistant on and off
- Define the AI connectors use by Elastic AI Assistant